### PR TITLE
Updated the styling of compact view to match new design sheets

### PR DESCRIPTION
**Pull request by chrhein**
Updated the styling for compact view tiles to match the new design sheets in Figma.

Before:
Screenshot 2021-07-14 at 10 55 15

After:
Screenshot 2021-07-14 at 10 52 50

### CODE REVIEW

File: src/dashboards/Compact/index.tsx [hunk 1]
```diff
@@ -73,16 +82,16 @@ const BREAKPOINTS = {
...
+    md: 2,
     sm: 1,
     xs: 1,
     xxs: 1,
 }
 
 const EnturDashboard = ({ history }: Props): JSX.Element | null => {
     const [settings] = useSettingsContext()
-    const [breakpoint, setBreakpoint] = useState<string>('lg')
+    const [breakpoint, setBreakpoint] = useState<string>(getDefaultBreakpoint)
```
Comment (draperunner) [inline src/dashboards/Compact/index.tsx:94]: Her må du kalle funksjonen: 
```diff
-useState<string>(getDefaultBreakpoint)
+useState<string>(getDefaultBreakpoint())
```

File: src/dashboards/Compact/styles.scss [hunk 2]
```diff
@@ -153,6 +118,21 @@
     }
 }
 
+@media (min-width: 996px) {
+    .compact {
+        &__tiles {
+            /*
+            react-grid-layout uses a margin of 32px between all grid-items.
```
Comment (draperunner) [inline src/dashboards/Compact/styles.scss:125]: Fint at du forklarar dette. 👍

Comment (anneliseostby) [reply src/dashboards/Compact/styles.scss:125]: .Kanskje du kan benytte rem i stedet for px:

Screenshot 2021-07-14 at 12 14 52

**Review by anneliseostby — APPROVED** [review]
Veldig fint 😊

**Review by fridasolheim — APPROVED** [review]

**Review by draperunner — APPROVED** [review]

**Pull request merged**